ide/coqide/preferences.ml

```diff
@@ -81,20 +82,18 @@ let stick (pref : 'a preference) (obj : < connect : #GObj.widget_signals ; .. >)
 
 (** Useful marshallers *)
 
-let mod_to_str m =
-  match m with
-    | `MOD1 -> "<Alt>"
-    | `MOD2 -> "<Mod2>"
-    | `MOD3 -> "<Mod3>"
-    | `MOD4 -> "<Mod4>"
-    | `MOD5 -> "<Mod5>"
-    | `CONTROL -> "<Control>"
-    | `SHIFT -> "<Shift>"
-    | `HYPER -> "<Hyper>"
-    | `META -> "<Meta>"
-    | `RELEASE -> ""
-    | `SUPER -> "<Super>"
-    |  `BUTTON1| `BUTTON2| `BUTTON3| `BUTTON4| `BUTTON5| `LOCK -> ""
+let mod_to_str = function
```

Comment:
Well, it’s better if we use `function`, isn’t it?

Reply:
IMO better to allow for small variations as long as they are easy to read.  Reformatting tends to obscure changes in function.  Everyone has slightly different formatting preferences, so it's the appearance is not going to be perfectly consistent.  From what I've seen, the code base is consistent in indenting the cases rather than putting them at the left margin.

Reply:
OK, I’ve put the indentation. But let’s keep `function`, it’s the keyword to use here.